src/proptypes/slice.js

```diff
@@ -18,6 +18,7 @@ const slicePropTypes = {
   fetchUTXOsError: PropTypes.string.isRequired,
   addressUsed: PropTypes.bool.isRequired,
   addressKnown: PropTypes.bool.isRequired,
+  lastUsed: PropTypes.string,
```

Comment:
This is optional but calculated with a selector (since it doesn't come back from client queries). It's used in the tables though so useful and necessary in prop checking.